py/test/test_DownloadArtifacts.py

```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+import json
+import unittest.mock as mock
+from unittest.mock import patch, mock_open
+import gzip
+
+from requests import Session
+
+path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
move this below imports.

Reply:
I need to set sys path before import parent directory, is there anyway I can do it? By set sys path through cli?